pre_commit_hooks/pretty_format_json.py

```diff
@@ -24,6 +24,25 @@ def _autofix(filename, new_contents):
         f.write(new_contents)
 
 
+def parse_indent(s):
+    # type: (str) -> str
+    try:
+        int_indentation_spec = int(s)
+        if int_indentation_spec >= 0:
+            return int_indentation_spec * ' '
+        else:
+            raise ValueError(
+                'Negative integer supplied to construct JSON indentation delimiter. ',
+            )
```

Comment:
Thanks, a `ValueError` was raised regardless, but the error message was misleading in case of invalid input. I have updated my branch. Can you revert the merge and remarge with the tip of my branch?